docs/v1/get-console-ids.md

```diff
@@ -50,7 +52,13 @@ const consoleIds = await getConsoleIds(authorization);
 
 ```json [HTTP Response]
 [
-  { "ID": 1, "Name": "Mega Drive" }
+  {
+    "ID": 1,
+    "Name": "Mega Drive",
+    "IconURL": "https://static.retroachievements.org/assets/images/system/md.png",
+    "Active": true,
+    "IsGameSystem": true
```

Comment:
Is this and the `a` and `g` params part of a different change? Can we break this out to a different PR -- I'd like to merge the `IconURL` change when we release to prod today.

Reply:
Ah, yes, this is coming in this [PR](https://github.com/RetroAchievements/RAWeb/pull/2266). 
I forgot this PR was still opened and my latest change was added, sorry.

Reply:
No worries, once the additions are reverted I'll immediately merge this branch (and it will deploy). We'll probably deploy RAWeb to prod in an hour or two.

Reply:
Just reverted